src/qt/bitcoingui.cpp

```diff
@@ -352,6 +357,8 @@ void BitcoinGUI::createActions(const NetworkStyle *networkStyle)
     openNetworkAction->setStatusTip(tr("Show network monitor"));
     openPeersAction = new QAction(QIcon(":/icons/connect_4"), tr("&Peers list"), this);
     openPeersAction->setStatusTip(tr("Show peers info"));
+    openRepairAction = new QAction(QIcon(":/icons/options"), tr("&Repair Wallet"), this);
```

Comment:
Every item in that list (before divider) is noun-based. I propose to keep that style here too i.e. change it to "Wallet &Repair" or "&Wallet Repair" or smth like that